docs/content/guides.mdx

```diff
@@ -1,16 +1,49 @@
+---
+title: Guides | Dagster Docs
+---
+
 # Guides
 
-## Dagster Guides
+Learn to accomplish common tasks in Dagster.
```

Comment:
I think losing the descriptions on this page might sacrifice more context than is worth it for the streamlining - each guide is a bird of such a different feather that seeing them in a list without additional context makes it difficult to get a handle on what's going on.

E.g. it's hard for me to imagine that someone coming to Dagster and seeing "Validating data with Dagster Type factories" would have any idea about what to do with it.

I also think several (most?) of these guides don't really fit the description "accomplish common tasks in Dagster":
- Enriching with Software-defined Assets - this is a guide on how to upgrade, not a guide on how to accomplish a task using Dagster
- Migrating to graphs, jobs, and ops - same as above
- Using the Hacker News example project - this is an overview of an example, not a task you'd accomplish

Reply:
That's a good point. This page was really tricky - I definitely didn't love it when I pushed this up the first time.

I took another shot at this. It doesn't use the two-column list like the other pages do, however - I opted for a single so summaries could be added without everything running together. At least with this approach (versus the original table), the elements are consistent from page to page, even if the columns are different.